$NetBSD: patch-bd,v 1.3 2011/12/24 17:07:07 dholland Exp $

Include <stdlib.h> and <time.h>, and disable the hand-written declarations
of random, srandom and time that they replace.  The hand-written ones say
time returns long, which is not what the header says, and having both is an
error rather than a duplicate.

--- src/fns.c.orig	1994-10-21 04:20:15.000000000 +0000
+++ src/fns.c
@@ -24,6 +24,8 @@ the Free Software Foundation, 675 Mass A
 /* 93.1.15  modified for Mule Ver.0.9.7.1 by Y.Akiba <akiba@cbs.canon.co.jp>
 	Patch for NeXT is updated. */
 
+#include <stdlib.h>
+#include <time.h>
 #include <config.h>
 
 /* Note on some machines this defines `vector' as a typedef,
@@ -64,7 +66,9 @@ With argument t, set the random number s
 {
   int val;
   unsigned long denominator;
+#if 0
   extern long random ();
   extern srandom ();
   extern long time ();
+#endif
 
   if (EQ (limit, Qt))
     srandom (getpid () + time (0));
